haystack/indexing/file_converters/docx.py

```diff
@@ -0,0 +1,28 @@
+from haystack.indexing.file_converters.base import BaseConverter
+import logging
+from pathlib import Path
+
+logger = logging.getLogger(__name__)
+
+class docxToTextConverter(BaseConverter):
```

Comment:
Please rename it to `DocxToTextConverter` to follow python naming conventions

Reply:
I've made this Change